cli/fs_util.rs

```diff
@@ -130,7 +130,10 @@ pub fn resolve_from_cwd(path: &Path) -> Result<PathBuf, AnyError> {
 /// Checks if the path has extension Deno supports.
 pub fn is_supported_ext(path: &Path) -> bool {
```

Comment:
Actually, I just realized this function name and looked at its usages. Perhaps we should remove `cjs` and `cts` from this function. For `is_supported_ext_fmt` I think it makes sense to have `cjs` and `cts`.

Reply:
Yes, though it would be weird supporting `cjs` & `cts` only in fmt and test, and not lint. just seems like it would be unexpected behaviour for someone that is using a `cjs` or `cts` fiel

Reply:
With compat mode only, this seems also weird, as to allow a certain filetype depending on compat just seems weird

Reply:
Sorry, my brain is fried at the moment after the swc upgrade. I meant for this and test to not enable it, but yeah I'm not sure what the desired behaviour is. Would be good to confirm with @bartlomieju before merging.

The one issue would be that picking up cjs/cts files would error for deno test if `--compat` mode wasn't enabled, but yeah maybe that's not a big deal and it would just be a reminder to turn on `--compat`.

Reply:
I think it's okay to always include `cjs` and `cts`